core/tests/ts-integration/tests/fees.test.ts

```diff
@@ -143,6 +179,113 @@ testFees('Test fees', () => {
         console.log(`Full report: \n\n${reports.join('\n\n')}`);
     });
 
+    test('Test gas price expected value', async () => {
+        const receiver = ethers.Wallet.createRandom().address;
+        const l1GasPrice = 2_000_000_000n; /// set to 2 gwei
+
+        await setInternalL1GasPrice(alice._providerL2(), {
+            newL1GasPrice: l1GasPrice.toString(),
+            newPubdataPrice: l1GasPrice.toString(),
+            customBaseToken: !isETHBasedChain
+        });
+
+        const receipt = await (
+            await alice.sendTransaction({
+                to: receiver,
+                value: BigInt(1)
+            })
+        ).wait();
+
+        const expectedETHGasPrice =
+            testMaster.environment().l1BatchCommitDataGeneratorMode === DataAvailabityMode.Rollup
+                ? 100_000_000 // 0.1 gwei (the minimum)
+                : 110_000_000; // 0.11 gwei, in validium we need to add compute overhead
+        const expectedCustomGasPrice = expectedETHGasPrice * 0.314;
```

Comment:
You shouldn't hardcode config values here, instead read them with `getFeeParams`

Reply:
`getFeeParams` will return you `FeeModelConfigV2` and `BaseTokenConversionRatio`, so you can replace hardcoded `100_000_000`, `110_000_000`, `0.314` with calculated values

Reply:
done